third_party/chromium/media/filters/source_buffer_stream.cc

```diff
@@ -2083,6 +2086,14 @@ base::TimeDelta SourceBufferStream::GetBufferedDurationForGarbageCollection()
   return duration;
 }
 
+void SourceBufferStream::EnableVideoBufferBudgetOverride() {
+  LOG(INFO) << "Enabling Video Buffer Budget Override. Current memory_limit_ is " << memory_limit_;
+  memory_limit_ = std::max(
```

Comment:
Will this happen on the same thread calling line 1907?  Otherwise there can be a race condition.

Reply:
this is now removed, so redundant.